implementations/federation-jvm/pom.xml

```diff
@@ -85,6 +85,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>cobertura-maven-plugin</artifactId>
+                <version>2.7</version>
```

Comment:
This shuts down a warning and makes builds reproducible. Without this, the `RELEASE` version would be used.